tests/python_functional/src/executors/tests/test_process_executor.py

```diff
@@ -24,11 +24,11 @@
 from src.executors.process_executor import ProcessExecutor
 
 
-def test_start_stop_process(tc_unittest):
+def test_start_stop_process(tmpdir):
     process_executor = ProcessExecutor()
-    process_command = ["python", "-c", "import time; time.sleep(3)"]
-    stdout_file = tc_unittest.get_temp_file()
-    stderr_file = tc_unittest.get_temp_file()
+    process_command = ["python", "-c", "import time; time.sleep(1)"]
+    stdout_file = tmpdir.join("stdout.log")
+    stderr_file = tmpdir.join("stderr.log")
```

Comment:
Is it necessary to set these? Stdout and stderr is not asserted.

Reply:
We have not assert it, but only passed to `process_executor.start()`  in the next line.

Reply:
It would be great if we could pass `None` instead of them.

Reply:
+1, I will fix this to None

Reply:
For now I would remain it as it is (using files in stdout/stderr). I have tried both dev/null and None, and with this file has been created in python_functional/ dir